src/js/components/message-indicator.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import DocumentTitle from 'react-document-title';
+
+const BLINKING_INTERVAL = 1500;
+
+export class MessageIndicatorComponent extends Component {
+    state = {
+        initialDocumentTitle: global.document ? document.title : '',
+        currentTitle: global.document ? document.title : ''
+    };
+
+    blinkTitle() {
+        if (!this.blinkInterval) {
+            const fn = () => {
+                const {unreadCount, messages, messageIndicatorTitle} = this.props;
+                const {currentTitle} = this.state;
+                let {initialDocumentTitle} = this.state;
+                
+                if (document.title !== initialDocumentTitle && !document.title.endsWith(messageIndicatorTitle.replace('{name}', ''))) {
```

Comment:
I'll add a reverse loop to start from the end of the conversation and get that name